src/adapters/adapter.utils.test.ts

```diff
@@ -0,0 +1,19 @@
+import { getAdapter } from './adapter.utils';
+import { AdapterEnum } from './adapter.types';
+import { requestReplyAdapter } from './requestReply.adapter';
+import { AdapterErrors } from './adapter.errors';
+
+describe('AdapterUtils', () => {
+  describe('getAdapter', () => {
+    it('should throw error', async () => {
+      expect(() => getAdapter('NonexistentAdapter')).toThrowError(AdapterErrors.AdapterNotFound);
```

Comment:
```suggestion
      await expect(getAdapter('NonexistentAdapter')).rejects.toThrowError(AdapterErrors.AdapterNotFound);
```

Reply:
I've tried your suggestion, but it fails... I'm not sure why...